searx/static/themes/simple/src/js/main/keyboard.js

```diff
@@ -63,6 +63,12 @@ searxng.ready(function () {
       des: 'remove focus from the focused input',
       cat: 'Control'
     },
+    'c': {
+      key: 'c',
+      fun: copyURLToClipboard,
+      des: 'copy URL of the selected result to the clipboard',
```

Comment:
```suggestion
      des: 'copy url of the selected result to the clipboard',
```
for consistency, lowercase is probably the better choice here too.